src/components/chat/w2wChat/TypeBar/Typebar.tsx

```diff
@@ -52,7 +52,7 @@ const Typebar = ({
   const isDarkMode = theme.scheme === 'dark';
 
   useEffect(() => {
-    if (textAreaRef) {
+    if (textAreaRef.current?.style) {
```

Comment:
make it like this textAreaRef?.current?.style

Reply:
Corrected @mishramonalisha76